src/store/dataset.ts

```diff
@@ -170,16 +172,18 @@ class DatasetStore {
   }
 
   removeCondition({ subGroup, itemName }: IRemoveConditionItem) {
-    const cloneConditions = cloneDeep(this.conditions)
+    const cloneConditions: Condition[] = cloneDeep(this.conditions)
```

Comment:
you don't need type assertion - cloneDeep returns a value of the same type as the initial value

Reply:
+